scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
Are you talking about in handlePet() where I see the same auto claim? (axe that one too)
Pro-tip: you can get links directly to a line by clicking the line number on the left

I don't recall what all hate mechanics these guys use, can't look it up atm (its *possible* updateEmntity may need to stay or super-linking happens, I am not the guy to answer that though), It doesn't sound like there would be much risk of someone depopping or otherwise losing them all though.

I just wanted to point out you aren't supposed to have the adds auto claimed. Normal claiming is you only hold 1 claim from one player on one mob, so additional party/alliance members would need to act to claim other mobs. And only PW himself will spawn pre-claimed (to the person that popped).